edx_event_bus_kafka/internal/producer.py

```diff
@@ -251,14 +308,10 @@ def poll_indefinitely(api_weakref: KafkaEventProducer):
         api_object = api_weakref()
         if api_object is None:
             return
-
         try:
             api_object.producer.poll(0)
         except BaseException:
-            # If polling is failing, we'll almost certainly find out about it from the poll call
-            # we make when producing an event. The call in this loop could be excessively noisy,
-            # so just debug-log it.
-            logger.debug("Event bus producer polling loop encountered exception (continuing)", exc_info=True)
+            logger.warning("Event bus producer polling loop encountered exception (continuing)", exc_info=True)
```

Comment:
1. Why is this not considered an error?
```suggestion
            logger.exception("Event bus producer polling loop encountered exception (continuing)")
```
2. Can we add a call to `record_exception()`? See other related comment.